mobile_files/package-lock.json

```diff
@@ -3486,6 +3486,468 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
+    "fsevents": {
```

Comment:
I think we sort of agreed not to have `fsevents` in package-lock, but can't remember actually, maybe best not to introduce it again

Reply:
@cammellos, I haven't heard about it. Can you elaborate a bit?
I thought that `package-lock.json` is always autogenerated and we just commit it `as is`. Am I wrong?

Reply:
@vkjr it's a long story :) https://github.com/npm/npm/issues/17722 
`fsevents` is a mac only optionaldependency and I believe used only on development ( but might be wrong). The problem is that linux users won't install it, so when re-generating package-lock they will remove the dependency and viceversa.

Reply:
@cammellos, so if I understand correctly we should remove "fsevents" section every time package-lock regenerated, but commit everything else, right?